Adiciona arquivo de script

JuanBarros2 opened the pull request:
Fix #38

FannyVieira commented on scripts/README.md:
```diff
@@ -0,0 +1,8 @@
+Esta pasta contém scripts para auxiliar no desenvolvimento
+de algumas funcionalidades.
+
+script                              | Objetivo
```
Acho que é bom adicionar uma coluna com os pré-requisitos

FannyVieira replied:
No caso,  pandas, numpy, e jupyter, podem ser links que redirecionam para a parte da documentação deles que explica a instalação

JuanBarros2 replied:
Então acho que não faz sentido deixar como tabela. Ficaria melhor criar sessões pra detalhar o script

FannyVieira replied:
Pode ser

FannyVieira commented:
Eu sinto que a gente precise fazer algo parecido com isso depois, ai acho que seria bom aplicar essas alterações passo a passo explicando o que acontece, ou só mostrando o resultado msm, tipo:

```
# Cria uma coluna para cada pré-requisito existente na lista de pré-requisitos
arestas = df.pre_requisitos.apply(lambda x: pd.Series(np.nan if str(x) == "-" else str(x).split(",")))
```

`
# ....
arestas = arestas.merge(df, right_index = True, left_index = True)
`
....
O que você acha?

JuanBarros2 commented:
> Eu sinto que a gente precise fazer algo parecido com isso depois, ai acho que seria bom aplicar essas alterações passo a passo explicando o que acontece, ou só mostrando o resultado msm, tipo:
> 
> ```python
> # Cria uma coluna para cada pré-requisito existente na lista de pré-requisitos
> arestas = df.pre_requisitos.apply(lambda x: pd.Series(np.nan if str(x) == "-" else str(x).split(",")))
> ```
> 
> ```python
> # Mescla as novas colunas geradas para as `arestas` ao `dataframe` com todos os dados das disciplinas
> arestas = arestas.merge(df, right_index = True, left_index = True)
> ```
> 
> ....
> O que você acha?



> Eu sinto que a gente precise fazer algo parecido com isso depois, ai acho que seria bom aplicar essas alterações passo a passo explicando o que acontece, ou só mostrando o resultado msm, tipo:
> 
> ```python
> # Cria uma coluna para cada pré-requisito existente na lista de pré-requisitos
> arestas = df.pre_requisitos.apply(lambda x: pd.Series(np.nan if str(x) == "-" else str(x).split(",")))
> ```
> 
> ```python
> # Mescla as novas colunas geradas para as `arestas` ao `dataframe` com todos os dados das disciplinas
> arestas = arestas.merge(df, right_index = True, left_index = True)
> ```
> 
> ....
> O que você acha?

Acho válido. Mas não sei muito como funciona o jupyter notebook nem como posso documentar direitinho um script python. Tu sabe?

FannyVieira commented:
> > Eu sinto que a gente precise fazer algo parecido com isso depois, ai acho que seria bom aplicar essas alterações passo a passo explicando o que acontece, ou só mostrando o resultado msm, tipo:
> > ```python
> > # Cria uma coluna para cada pré-requisito existente na lista de pré-requisitos
> > arestas = df.pre_requisitos.apply(lambda x: pd.Series(np.nan if str(x) == "-" else str(x).split(",")))
> > ```
> > 
> > 
> > ```python
> > # Mescla as novas colunas geradas para as `arestas` ao `dataframe` com todos os dados das disciplinas
> > arestas = arestas.merge(df, right_index = True, left_index = True)
> > ```
> > 
> > 
> > ....
> > O que você acha?
> 
> > Eu sinto que a gente precise fazer algo parecido com isso depois, ai acho que seria bom aplicar essas alterações passo a passo explicando o que acontece, ou só mostrando o resultado msm, tipo:
> > ```python
> > # Cria uma coluna para cada pré-requisito existente na lista de pré-requisitos
> > arestas = df.pre_requisitos.apply(lambda x: pd.Series(np.nan if str(x) == "-" else str(x).split(",")))
> > ```
> > 
> > 
> > ```python
> > # Mescla as novas colunas geradas para as `arestas` ao `dataframe` com todos os dados das disciplinas
> > arestas = arestas.merge(df, right_index = True, left_index = True)
> > ```
> > 
> > 
> > ....
> > O que você acha?
> 
> Acho válido. Mas não sei muito como funciona o jupyter notebook nem como posso documentar direitinho um script python. Tu sabe?

Acho que eu sei mais do jupyter notebook, de como documentar nn, eu posso procurar é bom que entendo melhor o que tu fez

JuanBarros2 commented:
@FannyVieira tu vai resolver isso dos comentários nessa branch ou vai criar uma issue para isso?

FannyVieira commented:
Eu vou merjar, e depois vou ajustar
